nntrainer/layers/lstm.cpp

```diff
@@ -31,16 +31,84 @@ enum LSTMParams {
   hidden_state,
   cell_state,
   ifgo,
+  reverse_weight_ih,
+  reverse_weight_hh,
+  reverse_bias_h,
+  reverse_bias_ih,
+  reverse_bias_hh,
+  reverse_hidden_state,
+  reverse_cell_state,
+  reverse_ifgo,
   dropout_mask
 };
 
+void batch_first_forwarding(
```

Comment:
make this function static, and doxygen for it.

Reply:
Done